src/index.js

```diff
@@ -98,6 +114,21 @@ class LinearRovingTabindex extends RovingTabindex {
     set wrap(newWrap) {
         this._navigationEmitter.model.options.wrap = newWrap;
     }
+
+    set _items(items) {
+        return items;
```

Comment:
This doesn't seem to set anything, is that correct?

Reply:
Correct. But the getter fails if there's no setter.

Reply:
The getter should not fail without a setter, there must be something else going on. 

Also again we should not need any getter or setter for `_items`, in this case we are already storing the selector on the instance and we just need a getter for `items`, eg, like I said:

```
get items() {
  return Util.nodeListToArray(this.el.querySelectorAll(this._itemSelector));
}
```

Reply:
Actually it was called `_items` before, so just make it `get _items` as you had, but with the code I have above.

Reply:
Did you see my update?

Reply:
Yeah, this is good: https://github.com/makeup-js/makeup-roving-tabindex/pull/1/files#diff-1fdf421c05c1140f6d71444ea2b27638R113

But we need to remove this: https://github.com/makeup-js/makeup-roving-tabindex/pull/1/files#diff-1fdf421c05c1140f6d71444ea2b27638R109

The error is because we are still assigning to this setter. We need to remove these lines:
https://github.com/makeup-js/makeup-roving-tabindex/blob/e9d91db34c8b0d2eb6c468567029ce1f44c185e0/src/index.js#L15
https://github.com/makeup-js/makeup-roving-tabindex/blob/e9d91db34c8b0d2eb6c468567029ce1f44c185e0/src/index.js#L96

Reply:
Removed.